### PR TITLE
Add episode 46

### PR DESCRIPTION
**Pull request by colebow**

### CODE REVIEW

File: _episodes/46.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+layout: episode
+title: "46: Trino heats up with Ignite"
+date: 2023-02-23
+tags: datbase memory in-memory ignite
```
Comment (mosabua) [inline _episodes/46.md:5]:
```suggestion
tags: database memory in-memory apache ignite
```

File: _episodes/46.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+youtube_id: ""
+wistia_id: ""
+sections:
+- title: "Trino Release"
```
Comment (mosabua) [inline _episodes/46.md:9]:
```suggestion
- title: "Trino releases"
```

File: _episodes/46.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+was added a couple releases ago in Trino 408. It's not every day that we add a
+new connector to Trino, and so the topic of today's episode is exploring the
+connector, what it does, and what its uses cases are. After that, we are going
+to talk about the process of coming in as an outside engineer and contributing 
+an entirely new connector to Trino.
+
+## What is Ignite?
+
+Apache Ignite is an in-memory distributed database, comparable to others you may
+be familiar with like SQL Server, Redis, and SingleStore. If you're not familiar
```
Comment (mosabua) [inline _episodes/46.md:76]: SQL Server ?? really .. I think of it as a traditional RDBMS

File: _episodes/46.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+with them or with in-memory computing, the gist is that by focusing on using RAM
+instead of disk storage, you can create a database system which is *much*
+faster - the Ignite website advertises 10-1000x improvements. Of course, this is
+more expensive, too, so it thrives in settings where performance is critical. 
+
+With an initial release 7 years ago, Ignite is still something of the new kid on
```
Comment (mosabua) [inline _episodes/46.md:82]: something of the new kid .... sounds weird

File: _episodes/46.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+the block among in-memory databases, coming with modern bells and whistles that
+has it positioned to become a successor to those other, comparable databases
+mentioned above. It also has some key functionality that sets it apart.
+
+## Ignite + Trino
```
Comment (mosabua) [inline _episodes/46.md:87]:
```suggestion
## Ignite and Trino
```

File: _episodes/46.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+## Contributing the Ignite connector
+
+The Trino community and developers try their best to be active reviewers,
+collaborators, and participants on pull requests coming in from outside
+contributors. Massive contributions like the Ignite connector can take a lot of
+round trips, back-and-forth discussion, and work from both the contributor and
+the project's maintainers to get it into a state where it is ready to merge and
+go live for users to try out.
+
+To give you an idea, the pull request (PR) to contribute Ignite was opened in
```
Comment (mosabua) [inline _episodes/46.md:100]: we should link to the PR

File: _episodes/46.md [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+## PR of the episode: #13943: Add support for `migrate` procedure in Iceberg
+
+If you've been in the data space for a while, you may know that there's a bit of
+a prevailing current in migrating from Hive to Iceberg. Out with the old, in
+with the new, and in with the performance gains. [Yuya Ebihara](https://github.com/ebyhr),
+one of the Trino maintainers, has added a table procedure to Trino's Iceberg
+connector to make that process much, much simpler. Rather than a slow, manual,
+and arduous process, if you have a Hive table stored in a file format supported
+by Iceberg, it's as simple as calling the `migrate` table procedure and letting
+it run. Neat, right?
```
Comment (mosabua) [inline _episodes/46.md:128]: Do we want to talk about what this does? Even high level  .. I assume it creates metadata in the Iceberg format, wipes most metadata in the HMS, and then updates the HMS to reference the iceberg format

File: _episodes/46.md [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+## More about Ignite
+
+* [Check out the Ignite website!](https://ignite.apache.org/)
+* [Ignite on Twitter](https://twitter.com/ApacheIgnite)
+* [Ignite on LinkedIn](https://www.linkedin.com/showcase/apache-ignite/)
+
+## Trino events
+
+If you have an event that is related to Trino, let us know so we can add it to
+the [Trino events calendar]({{site.url}}/community.html#events).
```
Comment (mosabua) [inline _episodes/46.md:139]: Call out Boston event

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _episodes/46.md [hunk 9]
```diff
@@ -0,0 +1,152 @@
+---
+layout: episode
+title: "46: Trino heats up with Ignite"
+date: 2023-02-23
```
Comment (chenjian2664) [inline _episodes/46.md:4]: Should be 2023-03-16?

File: _episodes/46.md [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* Manfred Moser, Director of Information Engineering at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* Jason (please comment however you'd like this to introduce you)
```
Comment (chenjian2664) [inline _episodes/46.md:39]: [Jason](https://www.linkedin.com/in/jian-chen-7aa3a2225) Senior data engineer at Shopee.

File: _episodes/46.md (hunk 9 above)
Comment (mosabua) [reply _episodes/46.md:4]: or 15 ...

**Review by bitsondatadev — APPROVED** [review]
image

**Pull request merged**